fvm/transaction_test.go

```diff
@@ -69,24 +69,25 @@ func TestTopShotSafety(t *testing.T) {
 		require.Equal(t, 0, buffer.Len())
 	})
 
-	t.Run("logs for failing tx but not related to TopShot", func(t *testing.T) {
-
-		runtimeError := runtime.Error{
-			Err: sema.CheckerError{
-				Errors: []error{&sema.ImportedProgramError{
-					CheckerError: &sema.CheckerError{},
-					ImportLocation: ast.AddressLocation{
-						Name:    "NotTopshot",
-						Address: common.BytesToAddress([]byte{0, 1, 2}),
-					},
-				}},
-			},
-		}
-
-		err, buffer := prepare(runtimeError)
-		require.Error(t, err)
-		require.Equal(t, 0, buffer.Len())
-	})
+	// We are now logging for all failed Tx
```

Comment:
Is this a temporary thing or can the code below be deleted?

Reply:
can probably be removed, will remove when i'm porting this over to master